flocker/route/functional/networktests.py

```diff
@@ -75,8 +75,8 @@ def test_some_proxies(self):
             proxy_two = self.network.create_proxy_to(ip, port + 1)
 
             self.assertEqual(
-                sorted([proxy_one, proxy_two]),
```

Comment:
Can you actual assume a specific return order? Why not sort them, or use `assertItemsEqual`?